test/functional/zmq_test.py

```diff
@@ -8,6 +8,7 @@
 import struct
 
 from test_framework.test_framework import BitcoinTestFramework, SkipTest
+from test_framework.mininode import CTransaction, BytesIO
```

Comment:
nit: BytesIO is taken from the python stl, so `from io import BytesIO` should work as well

Reply:
Done.